driver-core/src/test/functional/com/mongodb/ClusterFixture.java

```diff
@@ -140,7 +140,7 @@ public static boolean serverVersionLessThan(final List<Integer> versionArray) {
     }
 
     public static boolean serverVersionLessThan(final int majorVersion, final int minorVersion) {
-        return serverVersionAtLeast(asList(majorVersion, minorVersion, 0));
+        return serverVersionLessThan(asList(majorVersion, minorVersion, 0));
```

Comment:
Whoa, were we just not using this anywhere else?  If we are, better run a full patch build to make sure it doesn't break anything.

Reply:
I'll be sure to run a full patch build.